icons/chevron-down-12.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12"><path d="M6 8.8c-.2 0-.4-.1-.5-.2l-3-3c-.3-.3-.3-.8 0-1.1s.8-.3 1.1 0L6 6.9l2.5-2.5c.3-.3.8-.3 1.1 0s.3.8 0 1.1l-3 3c-.2.2-.4.3-.6.3z"/></svg>
```

Comment:
I think the build is failing because these SVGs don't have width and height attributes:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12" width="12" height="12"><path d="M6 8.8c-.2 0-.4-.1-.5-.2l-3-3c-.3-.3-.3-.8 0-1.1s.8-.3 1.1 0L6 6.9l2.5-2.5c.3-.3.8-.3 1.1 0s.3.8 0 1.1l-3 3c-.2.2-.4.3-.6.3z"/></svg>
```